addons/web/static/src/views/fields/properties/kanban_properties_field.xml

```diff
@@ -1,13 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
     <t t-name="web.KanbanPropertiesField" owl="1">
-        <div t-ref="properties" class="w-100">
+        <div t-ref="properties" class="w-100 mt-2">
             <div class="o_kanban_property_field d-inline-flex flex-column justify-content-center align-items-start w-100 my-1"
                 t-foreach="propertiesList"
                 t-as="propertyConfiguration"
                 t-key="propertyConfiguration.name"
-                t-if="(propertyConfiguration.value || ['integer', 'float'].includes(propertyConfiguration.type)) and propertyConfiguration.view_in_kanban">
-                <div class="mw-100 text-truncate">
+                t-if="propertyConfiguration.value &amp;&amp; propertyConfiguration.view_in_kanban">
+                <div class="mw-100 text-truncate d-flex gap-2"
+                    t-att-class="(['boolean'].includes(propertyConfiguration.type)) ? 'border border-dark rounded-3 ps-2' : ''">
```

Comment:
you can use dict directly
```suggestion
                    t-att-class="{propertyConfiguration.type === 'boolean': 'border border-dark rounded-3 ps-2'}">
```

(or inverse key / value, I never sure about the synthax)